benchmark_runner/main/environment_variables.py

```diff
@@ -31,6 +31,23 @@ def __init__(self):
         self._environment_variables_dict['elasticsearch'] = os.environ.get('ELASTICSEARCH', '')
         self._environment_variables_dict['elasticsearch_port'] = os.environ.get('ELASTICSEARCH_PORT', '')
 
+        # This parameter 'True'/'False' for debug workload, when its set to 'True' it will stop the workload when finishing
+        self._environment_variables_dict['stop_when_workload_finish'] = os.environ.get('STOP_WHEN_WORKLOAD_FINISH', '')
+
+        # default parameter - change only if needed
+        # Parameters below related to 'run_workload()'
+        self._environment_variables_dict['workloads'] = ['stressng_pod', 'stressng_vm', 'uperf_pod', 'uperf_vm', 'hammerdb_pod_mariadb', 'hammerdb_vm_mariadb',  'hammerdb_pod_postgres', 'hammerdb_vm_postgres', 'hammerdb_pod_mssql', 'hammerdb_vm_mssql']
+        self._environment_variables_dict['namespace'] = os.environ.get('NAMESPACE', 'benchmark-operator')
+        # run Hammerdb workload with ocs pvc True/False
+        self._environment_variables_dict['ocs_pvc'] = os.environ.get('OCS_PVC', 'True')
+        # This parameter get from Test_CI.yml file
+        self._environment_variables_dict['build_version'] = os.environ.get('BUILD_VERSION', '1.0.0')
```

Comment:
Isn't this also a constant?

Reply:
This is also something the user should never have to set.

Reply:
All I'm asking is to structure the file so that options the user might set are clearly separate from those a developer might change, so the file better documents itself.

Reply:
I have added this comment:
# run Hammerdb workload with ocs pvc True/False. True=OCS, False=Ephemeral